client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -36,7 +36,6 @@ function ReaderSubscriptionListItem( {
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
-	const siteUrl = url || getSiteUrl( { feed, site } );
```

Comment:
though if we wanted to be able to subscribe by a blog ID, and we wanted to stick a "pending" subscription into follow.items... hrm. though maybe we'd just assign a negative feed ID to mark it pending and then assign the proper one when it comes back.